roles/FortiManager-Admin-c57340f1-c387-474b-9c24-effc9e3c41aa.json

```diff
@@ -0,0 +1,585 @@
+{
+    "@context": "\/api\/3\/contexts\/Role",
+    "@id": "\/api\/3\/roles\/c57340f1-c387-474b-9c24-effc9e3c41aa",
+    "@type": "Role",
+    "description": "The FortiManager Admin should be able to edit only certain fields as the synchronization process owns many fields.",
+    "name": "FortiManager-Admin",
+    "modulePermissions": [
```

Comment:
@fortijames Please include only the custom modules that are added with this solution pack in the role. Exclude Out-of-the-Box (OOB) modules such as agents, alerts, and so on.